lib/defra_ruby_govpay/api.rb

```diff
@@ -22,49 +22,54 @@ def initialize(config = DefraRubyGovpay.configuration)
     end
 
     def send_request(method:, path:, params: nil, is_moto: false)
-      Rails.logger.debug build_log_message(method, path, params, is_moto)
+      @is_moto = is_moto
+      DefraRubyGovpay.logger.debug DefraRubyGovpay.logger.debug build_log_message(method, path, params)
```

Comment:
Not sure what this line suppose to do. Why DefraRubyGovpay.logger.debug added twice?
`DefraRubyGovpay.logger.debug DefraRubyGovpay.logger.debug build_log_message(method, path, params)`

Reply:
Copy and paste error, fixed